ci_framework/roles/artifacts/tasks/environment.yml

```diff
@@ -22,6 +22,7 @@
       find {{ cifmw_artifacts_basedir }}/artifacts/NetworkManager -type f -exec chmod 0644 '{}' \;
       find {{ cifmw_artifacts_basedir }}/artifacts/NetworkManager -type d -exec chmod 0755 '{}' \;
       test -d /etc/ci/env && cp -r /etc/ci/env {{ cifmw_artifacts_basedir }}/artifacts/ci-env
+      sudo test -d /var/log/bmaas_console_logs && sudo cp -r /var/log/bmaas_console_logs {{ cifmw_artifacts_basedir }}/logs
```

Comment:
`sudo` is not needed. It's running already as root: https://github.com/openstack-k8s-operators/ci-framework/pull/726/files#diff-5c8dde18abc01e76273f106544f0cdb251f60e4893720377a2736e37b787c533L17